kratos/geometries/nurbs_surface_geometry.h

```diff
@@ -254,6 +254,18 @@ class NurbsSurfaceGeometry : public Geometry<typename TContainerPointType::value
         return NumberOfKnotsV() - PolynomialDegreeV() + 1;
     }
 
+    /// returns number of control points in direction u.
+    SizeType GetNumberOfPointsU() const override
+    {
+        return NumberOfControlPointsU();
```

Comment:
It would make sense to rename `NumberOfControlPointsU/V` instead of heaving two functions with the same output.

Reply:
I agree. As this interface is already used I would change it as soon as this is agreed and merged.